app/models/job.rb

```diff
@@ -3,4 +3,7 @@ class Job < ActiveRecord::Base
   STATUSES = [:draft, :published, :archived]
   enumerize :state, in: STATUSES
   belongs_to :organization
+  belongs_to :published_by, foreign_key: :user_id, class_name: "User"
```

Comment:
Compelling argument. Now those 3 models `belongs_to` an 'author' instead of 'published_by'. 

Changes done ! :boom: